guides/team-and-project-scheduling

```diff
@@ -0,0 +1,67 @@
+
+### Scheduling work
+
+We use 10kft to schedule client work, across all three business units: delivery, strategy and technical operations. The scheduling team, a combination of sales, delivery and strategy team members, meet weekly to talk through team availability and the gaps we have in our schedule. Actions and decisions are noted in the #scheduling slack channel.
+
+*There are many aspects to scheduling, these predominantly are:*
+
+1.  future work we could win
+2.  current work being extended or amended
+3.  team shape changing
+
+Before the weekly scheduling meeting, 10kft should reflect:
+
+-   project needs: changes to existing projects
+-   sales needs: confirmed and potential new work
+
+Before each weekly meeting, the person responsible for the project or new sales lead ensures 10kft is updated. We use placeholders to reflect needs.
+
+During the meeting we use these placeholders to have discussions around the needs of the project: what the skill shape is and whether that matches up with the gaps in the schedule and the team members we have available.
+
+We base our scheduling conversation around placeholders. If a project or sales need isn’t added ahead of the scheduling meeting, it could be another week before needs are discussed.
+
+Tentative placeholders then get reassigned during the meeting once confirmed.
+
+### Sales opportunities - a bit more detail
+
+Tentative placeholders should be added to 10ft as soon as an opportunity reaches the pitch stage of a bid, to be discussed at scheduling.
+
+Each sales opportunity has a “sales lead”. This person is responsible for coming to scheduling to give the latest update and share the likelihood of a win, so we can assess against a project needing a similar skill set. This is compulsory. We undermine our own process if we do not have the latest information.
+
+The **sales lead** should:
+
+-   describe and explain the new opportunity in the #scheduling slack channel. Including nature of work, proposed team shape, possible scheduling challenges & potential start date
+-   add a tentative placeholder to 10kft. Go to the “How to…” sections for details of how to do this
+-   leave notes in scheduling channel with latest information
+  
+Once a sales opportunity is realised, this is handed over to the “project lead”, assigned to that work.
+
+The **project lead** should:
+
+-   ensure all tentative placeholders are accurate in 10kft    
+-   speak with the team assigned to the work to discuss next steps
+-   review and amend where needed the project set up in 10kft
+
+### Reporting our time
+
+Every team member working on a client or named project is responsible for reporting their time, as mentioned [here](http://playbook.dxw.com/#/?id=reporting-our-time) in the playbook. More on how to do this in the “how to” section.
```

Comment:
Links where the call to action is "here" aren't great from an accessibility point of view - I would make the call to action "in the playbook".